tests/BaseObjectTest.php

```diff
@@ -35,11 +39,13 @@ public function testGetApp()
 	/**
 	 * Test the setApp() function.
 	 * @return void
+	 *
+	 * @throws \Exception couldn't create app
 	 */
 	public function testSetApp()
 	{
 		$app = new App(__DIR__.'/../');
-		$this->assertNull($this->baseObject->setApp($app));
+		$this->baseObject->setApp($app);
```

Comment:
This may not be the most useful assertion, but it makes sure a setter does not return anything so why not keep it?

Reply:
I think there must be a cleaner solution to assert a void return.
There was already a discussion about it:
https://stackoverflow.com/questions/27511593/how-to-phpunit-test-a-method-with-no-return-value

Reply:
This discussion seems to be mostly about checking that the function had the correct effect and we already check that in the next assertion.
Checking that the function does not return anything was just a nice bonus, but why do you feel it isn't clean?